apps/extension/src/App/App.tsx

```diff
@@ -83,6 +84,11 @@ export const App: React.FC = () => {
     }
   }, [status, accounts]);
 
+  const parent = accounts.find(
+    (account) => account.type === KeyStoreType.Mnemonic
+  );
```

Comment:
Q: We have just single one with `KeyStoreType.Mnemonic`, right? Is that a direct `parent` to all derived ones?

Reply:
Technically we could have as many as we want, but right now, it's just the first one that gets created, so for now it's just one. This will be made a bit more robust if that need ever arises, which it may.